data/Philippines/House/sources/instructions.json

```diff
@@ -6,6 +6,21 @@
       "type": "membership"
     },
     {
+      "file": "morph/data.csv",
+      "create": {
+        "from": "morph",
+        "scraper": "everypolitician-scrapers/philippines-house-of-representatives",
+        "query": "SELECT * FROM data WHERE name != '' ORDER BY id"
+      },
+      "source": "http://www.congress.gov.ph",
+      "type": "membership",
+      "merge": {
+        "incoming_field": "name",
+        "existing_field": "name",
+        "reconciliation_file": "reconciliation/data.csv"
```

Comment:
You probably want to explain in the commit message why you're matching by `name` here, rather than by `id`